src/gtk/toga_gtk/widgets/context2d.py

```diff
@@ -0,0 +1,105 @@
+import cairo
```

Comment:
Does this call need to be wrapped in much the same way as the Webkit imports, to protect against platforms where (for whatever reason) Cairo isn't available?

Reply:
Yes I agree that this is a good idea. I committed a change to check for this using gi.require_foreign("cairo") which is the recommended method in the PyGObject user guide